controllers/fenceagentsremediation_controller.go

```diff
@@ -152,7 +154,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 		r.Log.Error(err, "Fence Agent response was a failure", "CR's Name", req.Name)
 		return emptyResult, err
 	}
-	if outputErr != "" || outputRes == "" {
+	if outputErr != "" || outputRes != SuccessFAResponse {
```

Comment:
I think we discussed this before, can't find it though: are we sure this is the response by all fence agents?
Can't we assume that `err == nil && outputErr == ""` mean success, no matter what `outputRes` is?

Reply:
We can, but I think it is more safe to look for this [success message](https://github.com/ClusterLabs/fence-agents/blob/main/lib/fencing.py.py#L1045) or the [return code zero](https://github.com/ClusterLabs/fence-agents/blob/main/lib/fencing.py.py#L1063).

Reply:
A non zero exit code will result in a non nil outputErr 🙂 
But also nice to see that all agents share the relevant code with the success message, thx 👍🏼

Reply:
YW it was a good question